### PR TITLE
Use different copy for self verification

### PR DESCRIPTION
**Pull request by dbkr**
The copy all refers to 'their' device for when you're verifying
someone else. Update it to refer to your own devices in the case
where you're verifying one of your own devices.

### Pull Request Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes (Hopefully not necessary for a string change...)
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)

Screenshot_2021-07-05-19-48-43-131_im vector app debug

Equivalent for iOS: https://github.com/vector-im/element-ios/pull/4525

### CODE REVIEW

**Review by bmarty — APPROVED** [review]
LGTM, thanks.
One remark: the "Can't scan" option is also a bit weird when self-verifying :/. We should maybe hide the subtitle in this case?

**Comment by dbkr** [comment]
oh, good point

**Comment by dbkr** [comment]
updated

File: vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt [hunk 1]
```diff
@@ -45,9 +45,22 @@ class VerificationChooseMethodController @Inject constructor(
         val host = this
 
         if (state.otherCanScanQrCode || state.otherCanShowQrCode) {
+            var scanCodeInstructions: String
+            var scanOtherCodeTitle: String
+            var compareEmojiSubtitle: String
+            if (state.isMe) {
+                scanCodeInstructions = host.stringProvider.getString(R.string.verification_scan_self_notice)
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt:52]: Sorry, I did not noticed it before, but there is no need to use `host.` here. It only useful if you want to use the host member from the Epoxy item DSL, for instance inside `bottomSheetVerificationNoticeItem {}` block.
It's not a big deal at all, it's OK to merge the PR like that, I can do the clean up later.

**Review by bmarty — APPROVED** [review]
Thanks for the update, LGTM

File: vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt:52]: (Fixed in 07d6eaacf95acfec165dce94e567ae89db8e3f14)

Comment (dbkr) [reply vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt:52]: ah - I see, I did wonder what that was all about, but figured it was best to be consistent anyhow

**Pull request merged**